openqa-investigate

```diff
@@ -283,7 +283,10 @@ post-investigate() {
     retry_result="$(echo "$job_data" | runjq -r '.job.result')" || return $?
     investigate_origin="$(echo "$job_data" | runjq -r '.job.settings.OPENQA_INVESTIGATE_ORIGIN')" || return 1
     origin_job_id=${investigate_origin#"$host_url/t"}
-    origin_job_data=$(client-get-job "$origin_job_id") || return $?
+    origin_job_data=$(client-get-job "$origin_job_id") || rc=$?
+    # shellcheck disable=SC2154
+    [[ $origin_job_data =~ $not_found_regex ]] && warn "Skip posting investigation comment on original job $origin_job_id as it does not exist anymore" && return 0
```

Comment:
```suggestion
    [[ $origin_job_data =~ $not_found_regex ]] && warn "Skipping posting investigation comment on original job $origin_job_id as it does not exist anymore" && return 0
```

but why warn and not just "echo"?

Reply:
To print it to stderr I suppose. I could also use echoerr if we have such a helper. However, I don't think considering this a warning is wrong because normally the original job is still supposed to be there. (And a warning is still not an error so it is in-line with returning 0.)

Reply:
There is nothing actionable, though? To my mind a warning means there's something to be sorted out. I don't see what one would be able to do here?

Reply:
Put another way if we wanted to ensure the original job was kept around we would have to add a comment to it before starting the investigation or some other means of keeping it around. Instead of a warning.